experimentalSetups/5-setup/hardware/motorControl/5-setup-mc.xml

```diff
@@ -47,10 +47,10 @@
         <param name="outputType">             pwm                   </param>
         <param name="fbkControlUnits">        metric_units          </param>
         <param name="outputControlUnits">     machine_units         </param>
-        <param name="kp">           500              </param>
+        <param name="kp">           -500              </param>
         <param name="kd">           0                </param>
-        <param name="ki">           100              </param>
-        <param name="maxOutput">    8000           </param>
+        <param name="ki">           -100              </param>
```

Comment:
@divyashah, are you sure about these negative signs?
Are they intended?

Reply:
Yes, we had to change the order of the motor phase wires on the 2FOC board, and to compensate for that we had changed the gains into negative.